server/src/test/java/org/elasticsearch/search/aggregations/bucket/nested/NestedAggregatorTests.java

```diff
@@ -470,7 +466,7 @@ public void testNestedOrdering() throws IOException {
                 nestedBuilder.subAggregation(maxAgg);
                 termsBuilder.subAggregation(nestedBuilder);
 
-                terms = searchAndReduce(new AggTestConfig(newSearcher(indexReader, false, true), termsBuilder, fieldType1, fieldType2));
+                terms = searchAndReduce(newSearcher(indexReader, false, true), aggTestConfig);
```

Comment:
I think `aggTestConfig = new AggTestConfig(termsBuilder, fieldType1, fieldType2);` needs to be added before this line? `termsBuilder` is changed in the lines above here, so a new `AggTestConfig ` instance should be created?

Reply:
yup, well spotted.